.travis.yml

```diff
@@ -1,11 +1,35 @@
+sudo: false
+
 language: node_js
+
 node_js:
-  - "0.12.7"
-  - "stable"
-services: mongodb
+  - 0.12.18
```

Comment:
Semver uses numbers, not strings, so `0.12.7` is before `0.12.18` so it is really a version *increase*. ;)